### DISCUSSION TITLE
Trilium server on FreeBSD?

### DISCUSSION THREAD

**Discussion by sydbarrett74** [TriliumNext/Trilium#1525] [Q&A]
Hello,

To what extent is the manual installation option supported for FreeBSD?

**Comment by zadam** [comment]
It's unsupported in the sense that I've never tried it.

I don't really see why it should not work though ...

**Comment by sydbarrett74** [comment]
Thank you for your quick reply, @zadam. 👍

**Comment by JIeJaitt** [comment]
@sydbarrett74 Have you tried deploying it on FreeBSD?
